util/tests/utiltst.t

```diff
@@ -2,7 +2,8 @@
 \l util/init.q
 
 np:.p.import[`numpy]
-
+ols:.p.import[`statsmodels.api]`:OLS
```

Comment:
Think these are left over from earlier ordering of the OLS/WLS in the utils folder?